_articles/send/send-lifespan.md

```diff
@@ -0,0 +1,209 @@
+
+Deletion Date has a **maximum allowed value of 30 days from creation**.
+
+### Deletion Behavior
+
+When a Send reaches its configured Deletion Date:
+
+- For recipients of the Send (i.e. anyone with the generated link), navigating to the Send link will show a screen reporting that the Send does not exist or is no longer available.
+
+- For the Sender, a {% icon fa-trash %} **Pending Deletion** icon will appear next to the Send. The Send will pend deletion for a few minutes, after which it will be permanently deleted from Bitwarden systems and from the Sender's Send view.
+
+  {% callout note %}There is no "soft delete" or Trash Can for Sends, meaning that once the pending deletion window has closed you will not be able to access the contents of a Send.{% endcallout %}
+
+## Expiration Date
+
+By default, Expiration Date will be set to Never. Using the **Expiration Date** option, you may change this to a range of pre-specified options (e.g. *1 hour*, *1 day*, *7 days*) or specify a custom timestamp (`MM/DD/YYYY HH:MM AM/PM` or `YYYY-MM-DD HH:MM` ([ISO 8601](https://en.wikipedia.org/wiki/ISO_8601){:target="\_blank"})).
+
+### Expiration Behavior
+
+When a Send reaches its specified Expiration Date:
+
+- For recipients of the Send (i.e. anyone with the generated link), navigating to the Send link will show a screen reporting that the Send does not exist or is no longer available.
+
+- For the Sender, an {% icon fa-clock-o %} **Expired** icon will appear next to the Send. The Send will remain accessible to its Sender in the Send view until the specified **Deletion Date** is reached.
+
+## Maximum Access Count
+
+For all Sends, a **Current Access Count** ticker will track the number of times the Send link has been accessed:
+
+{% image /send/send-access-ticker.png Current Access Count ticker %}
+
+For any Send, you may specify a **Maximum Access Count** that is a positive integer.
+
+### Maximum Access Count Behavior
+
+When A Send reaches its specified Maximum Access Count:
+
+- For recipients of the Send (i.e. anyone with the generated link), navigating to the Send link will show a screen reporting that the Send does not exist or is no longer available.
+
+- For the Sender, a {% icon fa-ban %} **Max access count reached** icon will appear next to the Send. The Send will remain accessible to its Sender in the Send view until the specified **Deletion Date** is reached.
+
+{% callout success %}
+The **Current Access Count** ticker counts the number of times the link has been accessed. It **does not** count the number of times the contents of a File Send are downloaded or the number of times the contents of a Text Send are copied.
```

Comment:
@fschillingeriv , you may want to double check with @MGibson1 , I believe this behavior now only counts access of file Sends upon download vs. viewing of the link itself.